docs/04-operation-guides/operations/02-install-kyma.md

```diff
@@ -2,25 +2,32 @@
 title: Install Kyma
 ---
 
-You can simply use the default Kyma installation, or modify it as it fits your purposes.
-Meet the prerequisites, provision a k3d cluster, and use the `deploy` command to run Kyma locally.
+You can simply use the default Kyma installation, or modify it as it fits your purposes:
+
+## Default installation
 
-## Prerequisites
+Meet the prerequisites, provision a k3d cluster, and use the `deploy` command to run Kyma locally.
 
-You have installed the following:
+### Prerequisites
```

Comment:
I disagree to putting the prerequisites below the default installation. I'm pretty sure the prerequisites apply to all non-default ways of installation too. The prerequisites and cluster provisioning should come first, and then all the different ways of installing. Actually, provisioning the cluster simply is one of the prerequisites, besides installing the CLI etc.

Reply:
Well, not exactly, as you don't need k3d and Docker to install it on a remote cluster. 
But anyway, I only updated a link there 
![image](https://user-images.githubusercontent.com/53601578/133633751-85425437-89e0-4620-8146-b8c1c093c1f9.png), so maybe talk to @mmitoraj about it? 'Cause it looks like this doc's structure needs a bit of love. There's no **Local installation** header, but there are two **Default installation** headers...